README.md

```diff
@@ -10,30 +9,49 @@ README
 What is RollerworksSearch?
 --------------------------
 
-RollerworksSearch provides you with a light yet powerful search system
-to easily integrate into your PHP application.
+RollerworksSearch provides you with a powerful search system
+for your PHP application.
 
 The system has a modular design and can work with any PHP framework,
 user locale, data format or storage system.
 
+Whether you want to simply search for users in your SQL database or want to
+provide a powerful search system for searching products using an ElasticSearch
+back-end, everything is possible.
+
+Say your customer data is stored in the "customer" table while the "invoice"
+data is stored in it's own table. And invoice rows also have there own table.
+
+Searching (using the FilterQuery syntax) can be as simple as:
+`invoice_price: >"$20.00" invoice_row: ~*"my cool product"; customer_type: !consumer.`
+
+You just searched in three relational tables! using a single condition with a
+user-friendly syntax. And that is just the start, checkout all the cool and
+powerful features RollerworksSearch has to offer you.
+
+**Note:** FilterQuery is just one of the supported input formats, you can use XML,
+JSON, PHP Array or simply build your own.
+
 Features
 --------
 
-Searches are performed using SearchConditions
-which can be easily exported to any format.
+RollerworksSearch provides you with most of the features you would expect
+from a search system. Including input processing, condition optimizing and
+transforming user input to a normalized data-format. And everything can be
+extended for your own use-cases and needs.
 
-A SearchConditions is build-up of fields and condition-groups.
-Each field can hold any type of value including ranges, comparisons
-and pattern matchers (starts/ends with contains and basic regex).
+Search conditions can be as simple or complex as you need them to be.
+Including grouping and nesting for the best possible result.
```

Comment:
this talks about many things but performance, i wonder if performance is impacted and how